ds3-sdk/src/main/java/com/spectralogic/ds3client/models/tape/TapeDrive.java

```diff
@@ -20,7 +20,6 @@
 
 import java.util.UUID;
 
-@JacksonXmlRootElement(namespace = "Data")
```

Comment:
Did you mean to remove this?  I'm not totally familiar with what this denotes, but the response data for GetTapeDrives is wrapped in a "Data" element at the top level (from self docs):

<Data><TapeDrive><ErrorMessage/><ForceTapeRemoval>false</ForceTapeRemoval><Id>462a2649-c3d2-46b8-9fe9-b514c76b16d2</Id><PartitionId>ecf64057-af05-4123-8412-cb437c9355c6</PartitionId><SerialNumber>test tape drive</SerialNumber><State>NORMAL</State><TapeId>ef158ef6-31d9-4549-9498-46ed52648c8a</TapeId><Type>UNKNOWN</Type></TapeDrive><TapeDrive><ErrorMessage/><ForceTapeRemoval>false</ForceTapeRemoval><Id>bfc45945-47c1-4294-a7f7-f22d04148b38</Id><PartitionId>ecf64057-af05-4123-8412-cb437c9355c6</PartitionId><SerialNumber>test tape drive 2</SerialNumber><State>NORMAL</State><TapeId/><Type>UNKNOWN</Type></TapeDrive></Data>